src/components/UI/DrawerItems/index.tsx

```diff
@@ -1,30 +1,138 @@
-import React from 'react';
-import ChevronLeftIcon from '@mui/icons-material/ChevronLeft';
-import { Box, IconButton, Divider } from '@mui/material';
+import * as React from 'react';
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import CssBaseline from '@mui/material/CssBaseline';
+import Divider from '@mui/material/Divider';
+import Drawer from '@mui/material/Drawer';
+import IconButton from '@mui/material/IconButton';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import MenuIcon from '@mui/icons-material/Menu';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+import MovieIcon from '@mui/icons-material/Movie';
+import BookIcon from '@mui/icons-material/Book';
+import StarRateIcon from '@mui/icons-material/StarRate';
 
-import './styles/index.scss';
+const drawerWidth = 240;
```

Comment:
constant variables like this shouldn't be camal cased lets turn this into all caps
ex. DRAWER_WIDTH